rdt-open-reader/src/main/res/layout/rdt_test_row.xml

```diff
@@ -8,15 +8,16 @@
     android:layout_marginLeft="10dp">
 
     <TextView
+        android:id="@+id/tests_row_rdt_id"
+        style="@style/CustomFontTextViewStyle.ListView.Bold.NoPadding"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:id="@+id/tests_row_rdt_id"
+        android:layout_marginStart="124dp"
         android:text="RDT ID 003"
         android:textColor="@color/black"
-        style="@style/CustomFontTextViewStyle.ListView.Bold.NoPadding"
         android:textSize="19sp"
-        app:layout_constraintTop_toTopOf="parent"
-        app:layout_constraintStart_toStartOf="parent" />
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
```

Comment:
The changes here will affect the existing app.

If needed, please create a different layout.